src/main/java/com/three_tech_solutions/slot_app/mappers/StudentMapper.java

```diff
@@ -14,24 +14,23 @@ public class StudentMapper {
      * El Plan y los Payments se reciben como parámetros, ya deben ser buscados o generados en el Service.
      */
 
-    public Student toStudent(StudentRequestDTO studentDTO, Plan plan, User user) {
+    public Student toStudent(CreateStudentRequest studentDTO, Plan plan, User user) {
         Student student = new Student();
-
-        student.setName(studentDTO.getName());
-        student.setLastname(studentDTO.getLastName());
-        student.setPhoneNumber(studentDTO.getCellphoneNumber());
-        student.setBirthday(studentDTO.getBirthday());
-        student.setAdmissionDate(studentDTO.getAdmissionDate());
-        student.setPathologies(studentDTO.getPathologies());
-        student.setEnabled(true);// Activo por defecto
-        student.setPlan(plan);
-        student.setUser(user);
-
-        return student;
+        return new Student(
+                studentDTO.getName(),
+                studentDTO.getLastName(),
+                studentDTO.getCellphoneNumber(),
+                studentDTO.getBirthday(),
+                studentDTO.getPathologies(),
+                studentDTO.getAdmissionDate(),
+                true,
+                plan,
+                user
+        );
     }
 
-    public StudentResponseDTO toStudentDTO(Student student) {
-        StudentResponseDTO studentDTO = new StudentResponseDTO();
+    public StudentResponse toStudentResponseDTO(Student student) {
```

Comment:
Acá no hace falta que agreguemos "DTO"

```suggestion
    public StudentResponse toStudentResponse(Student student) {
```